docs/QuickStart.md

```diff
@@ -16,8 +16,8 @@ $ sbt dist
 Once project packaging is completed, unzip archive created in path : `package`.
 ```bash
 $ cd package
-$ unzip nsdb-0.6.0-SNAPSHOT.zip
-$ cd nsdb-0.6.0-SNAPSHOT/bin
+$ unzip nsdb-1.0.0-SNAPSHOT.zip
+$ cd nsdb-1.0.0-SNAPSHOT/bin
 $ ./nsdb-cluster
 ```
 In order to check if the application is up and running properly user can call health-check API:
```

Comment:
Maybe it would be useful to add the way to run NSDb using the docker image?

Something like

`docker run -it -p 9000:9000 -p 7817:7817 -d --name nsdb_docker  weareradicalbit/nsdb:1.0.0-SNAPSHOT`

and the cli too

`docker exec -it nsdb_docker bin/nsdb-cli --host 127.0.0.1 --database <database_name>`

Reply:
In this file even if it's not pointed out here in the PR, a brief description of how to run NSDb from a docker-compose has been made.  you can have a look at it from the source and see if it's comprehensive

Reply:
Oh yes, I see.

I think we have to change this

`$ docker run --rm -it tools.radicalbit.io/nsdb:0.6.0-SNAPSHOT bin/nsdb-cli --host %HOST_IP% --port 7817 --database database_name`

into 

`$ docker run --rm -it weareradicalbit/nsdb:1.0.0-SNAPSHOT bin/nsdb-cli --host %HOST_IP% --port 7817 --database database_name`

Reply:
Ops... Missed it. Thanks, I'll fix it!